docs/auto-tag-orders-with-utm-parameters/script.liquid

```diff
@@ -15,15 +15,19 @@
         id
         name
         tags
-        customerJourney {
-          moments {
-            ... on CustomerVisit {
-              utmParameters {
-                campaign
-                content
-                medium
-                source
-                term
+        customerJourneySummary {
+          moments (first: 250) {
```

Comment:
tiny adjustment to match [conventional use](https://graphql.org/learn/queries/): use `field(args)` instead of `field (args)`